client/main.go

```diff
@@ -15,54 +15,27 @@ import (
 	"google.golang.org/grpc/credentials"
 )
 
+// New returns a new configured cacher client for the given facility.
 func New(facility string) (cacher.CacherClient, error) {
-	lookupAuthority := func(service, facility string) (string, error) {
-		_, addrs, err := net.LookupSRV(service, "tcp", "cacher."+facility+".packet.net")
-		if err != nil {
-			return "", errors.Wrap(err, "lookup srv record")
-		}
-
-		if len(addrs) < 1 {
-			return "", errors.Errorf("empty responses from _%s._tcp SRV look up", service)
-		}
-
-		return fmt.Sprintf("%s:%d", strings.TrimSuffix(addrs[0].Target, "."), addrs[0].Port), nil
-	}
-	var securityOption grpc.DialOption
```

Comment:
Oh fun, I did this same change in #24. Whichever one merges first the other will have to fix up. I don't mind either way.